AutomatedTesting/Passes/MainPipeline.pass

```diff
@@ -42,6 +42,11 @@
                         "RayTracingAccelerationStructurePass"
                     ]
                 },
+                {
+                    "Name": "ClipmapGenerationPass",
+                    "TemplateName": "ClipmapGenerationPassTemplate",
+                    "Enabled": true
+                },
```

Comment:
Do we have a separate Jira ticket for dealing with pass additions? Right now it looks like this will need to get added to every project that has terrain enabled (like AtomTechArt for example, and any customer projects out there, etc...).  This will need some sort of Impactful Change e-mail I think, if existing terrain projects will soon break without it.

Reply:
Yes, you are right. It's LYN-8602. And the Atom team is working on adding a pass to the pipeline dynamically based on whether a gem is enabled.
Here "Enabled" should be false. I did some final test and forgot to change it back.